calculatorUICOOP/Models/MathLogic.cs

```diff
@@ -33,5 +33,10 @@ public static double Divide(double num1, double num2)
         {
             return num1 / num2;
         }
+
+        public static double ConvertToPercent(double num1)
+        {
+            return num1 / 100;
```

Comment:
Not sure why logic is implemented in this change, but since it's been open a while, that's fine.

Reply:
Fixed, thanks for the review.